.changeset/tidy-guests-do.md

```diff
@@ -0,0 +1,5 @@
+---
+'@talend/react-components': major
```

Comment:
fix is patch

Reply:
Might break some cases where it's already used, so I was hesitating